src/app-check/app-check-api-client-internal.ts

```diff
@@ -147,9 +147,9 @@ export class AppCheckApiClient {
    */
   private toAppCheckToken(resp: HttpResponse): AppCheckToken {
     const token = resp.data.attestationToken;
-    // `timeToLive` is a string with the suffix "s" preceded by the number of seconds,
+    // `ttl` is a string with the suffix "s" preceded by the number of seconds,
```

Comment:
Interesting -- unrelated to this review, the `ttl` is actually a textual representation of a `Duration` proto as per [AIP-142](https://google.aip.dev/142#durations) and [the `Duration` documentation](https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/duration.proto). The only downside is that parsing it via `Duration` might cost us an additional dependency.

Reply:
Yeah. I think for Javascript (Node.js) we can parse the JSON by referring to the JSON Mapping section in the proto. Can confirm that the JS SDK follows the same approach.